.circleci/config.yml

```diff
@@ -268,6 +271,123 @@ jobs:
+          command: mkdir -p libs/gutenberg-mobile/gutenberg/packages/react-native-bridge/android/src/main/assets
+      - attach_workspace:
+          at: libs/gutenberg-mobile/gutenberg/packages/react-native-bridge/android/src/main/assets
+      # - run:
+      #     name: Build
+      #     environment:
+      #       SUPPRESS_GUTENBERG_MOBILE_JS_BUNDLE_BUILD: 1
+      #     command: ./gradlew WordPress:assembleVanillaDebug WordPress:assembleVanillaDebugAndroidTest --stacktrace
+      - run:
+          name: Decrypt credentials
+          command: openssl aes-256-cbc -md sha256 -d -in .circleci/.firebase.secrets.json.enc -out .circleci/.firebase.secrets.json -k "${FIREBASE_SECRETS_ENCRYPTION_KEY}"
+      # - android/firebase-test:
+      #     key-file: .circleci/.firebase.secrets.json
+      #     type: instrumentation
+      #     apk-path: WordPress/build/outputs/apk/vanilla/debug/org.wordpress.android-vanilla-debug.apk
+      #     test-apk-path: WordPress/build/outputs/apk/androidTest/vanilla/debug/org.wordpress.android-vanilla-debug-androidTest.apk
+      #     test-targets: package org.wordpress.android.ui.screenshots
+      #     devices: |
+      #       model=blueline,version=28,orientation=portrait
+      #       model=gts3lltevzw,version=28,orientation=default
+      #     # [Trial Limitation] Free plan I use to experiment with Firebase Test Lab has a quota of 10 tests/day
+      #     # locales: ar de_DE en_US es_ES fr_FR iw_IL in it_IT ja_JP ko_KR nl_NL pl_PL pt_BR ru_RU sr sv_SE th tr_TR vi zh_CN zh_TW
+      #     locales: en_US in
+      #     project: wpandroid-74b6c
+      #     timeout: 30m
+      #     num-flaky-test-attempts: 2
+      #     results-history-name: CircleCI WordPress Screenshots
+      # - android/save-gradle-cache
+
+      # BEGIN DEBUG STEP (While firebase-test is commented out)
+      - run:
+          name: Fake FTL Run
+          command: |
+            gcloud auth activate-service-account --key-file ".circleci/.firebase.secrets.json"
+            echo "INFO: Raw results root path is: [gs://test-lab-d1urt75q3y6wu-ha5s1mih364hi/2020-08-26_08:51:14.975190_umlG/]" >log.txt
+      # END DEBUG STEP
+
+      - run:
+          name: Download Raw Screenshots from Google Storage
+          command: |
+            TEST_BUCKET=$(cat log.txt | grep -o "gs://test\-lab\-.*/" | head -1)
+            bundle exec fastlane download_raw_screenshots bucket:"$TEST_BUCKET" phone:blueline-28 tenInch:gts3lltevzw-28
+      - persist_to_workspace:
+          root: .
+          paths:
+            - Gemfile
+            - Gemfile.lock
+            - fastlane/
+  promo-screenshots:
+    parameters:
+      post-to-slack:
+        description: Post to Slack when screenshots job has finished. SLACK_WEBHOOK ENV variable must be set.
+        type: boolean
+        default: true
+    macos:
+      xcode: 11.6.0
+    working_directory: /tmp/workspace
+    steps:
+      - attach_workspace:
+          at: .
```

Comment:
This allows me to optimise CI steps, by only storing, at the end of the `promo-screenshots` job (L333-338, `persist_to_workspace`), the bare minimum required to create promo screenshots, and restoring just those here, instead of re-cloning the entire git repo needlessly.